src/main/resources/hazelcast-config-schema.json

```diff
@@ -1358,6 +1334,59 @@
           }
         }
       }
+    },
+    "BackupCount": {
+      "description": "Number of synchronous backups. For example, if 1 is set as the backup count, then all entries of the map will be copied to another JVM for fail-safety. 0 means no sync backup. The sum of backup-count and async-backup-count can't be larger than than 6.",
+      "type": "integer",
+      "minimum": 0,
```

Comment:
it’s not really a comment, but maybe we can organise values in some order for better readable experience:
first version:
```
            "default": 0,
             "minimum": 0,
             "maximum": 6,
```
updated:
```
      "minimum": 0,
       "default": 1,
       "maximum": 6
```